roxie/ccd/ccdserver.cpp

```diff
@@ -12360,7 +12360,9 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
             buildUserMetadata(metadata);
             buildLayoutMetadata(metadata);
             unsigned nodeSize = metadata->getPropInt("_nodeSize", NODESIZE);
-            if (metadata->getPropBool("_noSeek", false))
+            //MORE: Could be cached in the factory.
```

Comment:
I think you need to do so - queryWorkUnit will return NULL for predeployed queries I think.

Reply:
see QueryOptions class in ccdquery for the way this sort of thing is normally done